internal/dashboard/user_handler.go

```diff
@@ -91,6 +92,17 @@ func (s *Server) DeleteUser(ctx context.Context, userId string) (dashv1alpha1.Im
 	log := clog.FromContext(ctx).WithCaller()
 	log.Debug().Info("request", "userId", userId)
 
+	caller := callerFromContext(ctx)
+	if caller == nil {
+		err := kosmo.NewInternalServerError("user is not found in context", nil)
+		return ErrorResponse(log, err)
+	}
+
+	if userId == caller.Name {
+		err := kosmo.NewForbiddenError("trying to delete own user", nil)
```

Comment:
more fluent

```suggestion
		err := kosmo.NewForbiddenError("trying to delete yourself", nil)
```

Reply:
Which does deleting themselves mean Forbidden request  or Bad request ?
I think it is Bad request and feel appropriate to deny with 400

Reply:
thanks. I fixed it.